client/my-sites/posts/style.scss

```diff
@@ -619,6 +623,7 @@
 	flex-grow: 1;
 	margin: 0 0 24px 24px;
 	display: none;
+	max-width: 400px;
```

Comment:
For the 1600px breakpoint, something like 640 for the drafts panel's max-width seems better.